src/docfx/restore/FileResolver.cs

```diff
@@ -50,10 +60,16 @@ public Stream ReadStream(SourceInfo<string> file)
             if (!UrlUtility.IsHttp(file))
             {
                 var localFilePath = Path.Combine(_docsetPath, file);
+                FilePath fallbackFilePath;
                 if (File.Exists(localFilePath))
                 {
                     return File.OpenRead(localFilePath);
                 }
+                else if (_fallbackDocset != null && _input != null
```

Comment:
`Path.Combine(_fallbackDocset.DocsetPath, file)` already produced the full path, why not use it directly w/o Input?

The original intention of `FilePath` is to been able to construct it without using absolute path (maybe it needs a check in the constructor), so you can `new FilePath("a.md", Origin.Fallback)`.

Reply:
Good point, we only have files in the fallback docset, just a different local file path.
Adding a check for `FilePath`

Reply:
the check would cause more code change, fix it in a new PR instead